src/OSPSuite.Core/Domain/Builder/BuildingBlock.cs

```diff
@@ -64,7 +64,13 @@ public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager clone
       }
    }
 
-   public interface IBuildingBlock<TBuilder> : IBuildingBlock, IEnumerable<TBuilder> where TBuilder : class, IObjectBase
+   public interface IBuilder : IObjectBase
+   {
+      //Reference to building block containing this entity. This does not have to be serialized
+      IBuildingBlock BuildingBlock { get; set; }
```

Comment:
THe Ibuilder needs the reference to that we can jump to the correct builder if something is wrong (in MoBi)